docs/src/dev-manual/devel.md

```diff
@@ -1,10 +1,15 @@
 # Local/Development environment
 
-Enduro uses Tilt to set up a local environment building the Docker images in a
-Kubernetes cluster. It has been tested with k3d, Minikube and Kind.
+## Quickstart with Devbox
+
+First install [devbox]. After installing devbox, run `devbox shell`. Continue
+the instructions [here](#dex-host).
```

Comment:
Can you make this a subsection of Requirements and briefly describe what devbox is and that it's experimental and not part of the standard dev workflow yet?